sdk/daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Runner.scala

```diff
@@ -221,7 +221,7 @@ object Script {
 
   final case class FailedCmd(description: String, stackTrace: StackTrace, cause: Throwable)
       extends RuntimeException(
-        s"""Command ${description} failed: ${cause.getMessage}
+        s"""Command ${description} failed: ${Option(cause.getMessage).getOrElse(cause)}
```

Comment:
The ScriptRunner output changes are caused by this, they're getting better output now.
The behaviour is correct though, dont worry :)